app/services/vendor_api/single_application_presenter.rb

```diff
@@ -61,5 +61,11 @@ def get_rejection
         }
       end
     end
+
+    def get_offer
```

Comment:
Should this still return `nil` if there's no offer?

Reply:
Yes. Would have to check the ApplicationChoice status as you can still have an offer without conditions :/

Reply:
Updated here: d59a1ff049952e71b2694bcc6c28b08d5dfdab1c
What do you think?

Reply:
What about when the ApplicationChoice is in a state like `conditions_met` — the offer should still exist?

Reply:
Ooh good catch @duncanjbrown.